src/liboslexec/oslexec_pvt.h

```diff
@@ -594,7 +599,8 @@ class ShaderInstance {
 
     /// Does it appear that the layer is completely unused?
     ///
-    bool unused () const { return run_lazily() && ! outgoing_connections(); }
+    bool unused () const { return run_lazily() && ! outgoing_connections()
```

Comment:
I think you are right. I don't mind that it's explicit here, and I can imagine a future in which we separate the concepts a little more. But if you think this detracts from understanding of the code, I can remove that clause and rely on run_lazily().